linters/javascript/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "rules": {
+    "consistent-return": 0,
+    "id-length": [2, {"exceptions": ["_", "x", "y"]}],
```

Comment:
may I ask what this is?
we can't have variables with one character, unless they're x or y?